README.md

```diff
@@ -1 +1,194 @@
+
+- **[Fully Tested `Rspec & Capybara`]** The app is fully tested with Rspec & Capybara, ensuring reliability and a seamless financial management experience on any device.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo & preview <a name="live-demo"></a>
+
+
+https://github.com/VelzckC0D3/Budget_App/assets/92229666/e7ecc547-00bc-4d1a-9f25-f7dd07ba93f3
+
+- _You can visit the live demo [here](https://rails-eb2s.onrender.com/)_
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ To have Ruby on Rails setted up in your local machine.
+```
+
+```sh
+ To have PostgreSQL setted up in your local machine.
+```
+
+```sh
+ To have a browser to run the project.
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+Open it with Visual Studio Code (or your preffered IDE), and open a server with "Rails -s".
+```
+
+```sh
+Open your desired browser and go to "localhost:3000".
+```
+
+### Install
+
+Install this project with:
+
+```sh
+bundle install
+
+yarn install --check-files
+
+rails db:create
+
+rails db:migrate
+
+rails db:seed
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+rails server
+```
+
+### Deployment
+
+This project is already fully deployed and ready to use, but if you want to deploy it again, follow the steps marked on the official render guide [here](https://render.com/docs/deploy-rails)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+- GitHub: [@VelzckC0D3](https://github.com/VelzckC0D3)
+- LinkedIn: [VelzckC0D3](https://www.linkedin.com/in/velzckcode/)
+
+<!-- FEATURES -->
+
+## 🔭 Future Features <a name="features"></a>
+
+- **[User Confirmation]** Users will be able to confirm their accounts with a confirmation email.
+- **[User Password Reset]** Users will be able to reset their passwords with a reset password email.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, be pending on my profile since I'll be doing much more!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thanks my Microverse Team and partners for helping me to get this done.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
```

Comment:
- [ ] according to the requirements The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

Reply:
@mikethreels oh my... i didn't notice, thank you so much 😅